.template-lintrc.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 module.exports = {
-  extends: 'recommended'
+  extends: 'octane'
```

Comment:
Don't know the implication of this, e.g. I don't know if this will impact consuming applications, but probably not.

Reply:
This should only affect template linting for this addon.